include/lsst/meas/extensions/trailedSources/VeresModel.h

```diff
@@ -0,0 +1,100 @@
+// -*- lsst-c++ -*-
+/*
+ * This file is part of meas_extensions_trailedSources.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef LSST_MEAS_EXTENSIONS_TRAILEDSOURCES_VERES_H
+#define LSST_MEAS_EXTENSIONS_TRAILEDSOURCES_VERES_H
+
+#include "lsst/pex/config.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/geom.h"
+#include "lsst/geom.h"
+#include "ndarray.h"
+
+namespace lsst {
+namespace meas {
+namespace extensions {
+namespace trailedSources {
+
+/**
+ * Implementation of an axisymmetric 2D Gaussian convolved with a line -- a model
+ * for a fast-moving, trailed-source (Veres et al. 2012).
+ *
+ * VeresModel is designed to compute the chi-squared of the model given an
+ * `lsst::afw::image::Exposure`, and to be passed to scipy.optimize for minimization.
+ */
+class VeresModel {
+public:
+    typedef afw::image::Image<float> Image;
+    typedef afw::image::Exposure<float> Exposure;
```

Comment:
Assuming these are intended as shorthands and not as abstractions, consider calling them `ImageF` and `ExposureF`, per [existing Pipelines convention](https://developer.lsst.io/cpp/style.html#a-a-name-representing-a-typedef-should-have-a-t-suffix-if-and-only-if-necessary-to-disambiguate-the-typedef-from-a-template-bare-name).

If these are shorthands, it might be worth making them private to avoid cluttering up the class documentation. However, I'm not sure what that would do the the docs for the method signatures.

Reply:
The use of `typedef` is discouraged as of C++11; prefer `using` (I'll admit the style guide is not up to date on this):
```suggestion
    using Image = afw::image::Image<float>;
    using Exposure = afw::image::Exposure<float>;
```